tox.ini

```diff
@@ -0,0 +1,52 @@
+[tox]
+envlist = py3, pycodestyle, pyflakes, pylint
+recreate = True
+
+[testenv]
+basepython = python3.6
+setenv =
+    FAKE_TRANSLATE = always
+deps =
+    urwid==1.2.1
+    PyYAML
+    attrs
+    systemd-python
+    jsonschema
+    pyudev
+    requests
+    requests-unixsocket
+    --editable=git+https://github.com/CanonicalLtd/probert@670c7258014eef13a971be0bf12206d96f2829f3#egg=probert
+
+
+[testenv:pycodestyle]
+deps =
+    pycodestyle==2.4.0
+commands = {envpython} -m pycodestyle \
+    {posargs:subiquity/ subiquitycore/ console_conf/}
+
+[testenv:pylint]
+deps =
+    {[testenv]deps}
+    pylint==1.9.1
+
+[testenv:py3]
+deps =
+    {[testenv]deps}
+    nose
+
+commands = {envpython} -m nose \
+    {posargs:subiquity/ subiquitycore/ console_conf/}
+
+
+[testenv:coverage]
+deps = {[testenv:py3]deps}
+    nose-timer
+    coverage
+commands = {envpython} -m nose \
+    --with-timer --timer-top-n 10 \
+    --with-coverage --cover-erase --cover-branches --cover-inclusive \
+    {posargs:subiquity/ subiquitycore/ console_conf/}
+
+[flake8]
```

Comment:
is this copy + paste from another tox? not even invoked, so is this to have around? Without a command and dep on flake8 does this serve a purpose?

Reply:
not copied. i specifically added this section, and I think its nice.
coverage is significantly slower than unit tests, so generally when a developer is running unit tests or writing them they do not want coverage reports. but when they are trying to increase coverage, they do wan that.  so having 2 things i think is nice.

if we're running from c-i i guess we would want the coverage in to run by default and have the 'py3' bare just present for developer usage.

Reply:
I was asking about the flake8 section not the coverage; no issues with coverage ;)